cmd/server-main.go

```diff
@@ -984,7 +984,7 @@ func serverMain(ctx *cli.Context) {
 
 		// Initialize site replication manager after bucket metadata
 		bootstrapTrace("globalSiteReplicationSys.Init", func() {
-			globalSiteReplicationSys.Init(GlobalContext, newObject)
+			go globalSiteReplicationSys.Init(GlobalContext, newObject)
```

Comment:
Why is it a goroutine?

Reply:
not needed - this is already being. called in a goroutine